pubspec.yaml

```diff
@@ -80,7 +82,7 @@ flutter_icons:
 
 
 flutter:
-
+  generate: true

```

Comment:
what is this for?

Reply:
it is used to generate the localization files